sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -139,9 +139,48 @@ Applications must make sure that the Level-Zero handles themselves aren't used s
 Practically speaking, and taking into account that SYCL runtime takes ownership of the Level-Zero handles,
 the application should not attempt further direct use of those handles.
 
+## 5 Level-Zero additional functionality
+
+### 5.1 Free Memory query
+Level Zero contains a library for system resource management called Sysman.  
+Sysman allows programmers to query the amount of free, or unallocated, 
+memory in a device.  This extension provides a new device information
+descriptor specific to Level-Zero Backends that allows DPC++ programs to query
+the amount of free memory for a given device. The new descriptor is as follows: 

```

Comment:
```suggestion
The Level Zero backend provides the following device information descriptors 
that an application can use to query information about a Level Zero device.
Applications use these queries via the `device::get_backend_info<>()` member 
function as shown in the example below (which illustrates the `free_memory`
query):
```

Let's word the introduction to this section in a general way that lets us document more info descriptors in the future.  I don't think it's necessary to document the fact that some of these descriptors come from the Sysman library.  A DPC++ user doesn't need to know that.

You can follow that introduction with an example like you have below:

``` C++
sycl::queue Queue;
auto Device = Queue.get_device();
size_t freeMemory = Device.get_backend_info<sycl::ext::oneapi::level_zero::info::device::free_memory>();
```

(Sorry, GitHub wouldn't let me add a code sample to my "suggestion".)